File: PRAK502-2310817210001-AndraBraputraAkbarSaleh.c
#include <stdio.h>
#include <math.h>
int hitung(int nilai1, int nilai2){
    int rumus = (nilai1 - nilai2);
    return rumus;
}
int mutlak(int angka){
    int nilaimutlak = fabs(angka);
    return nilaimutlak;
}
int main(){
 int a,b,c,d;
 scanf("%d %d %d %d", &a, &b, &c, &d);
 int Hasil = mutlak(hitung(a,c)) + mutlak(hitung(b,d));
 printf("%d",mutlak(Hasil));
 return 0;
}

File: PRAK502-2310817210001-AndraBraputraAkbarSaleh.py
from math import fabs

def hitung(nilai1, nilai2):
    rumus = nilai1 - nilai2
    return rumus

def mutlak(angka):
    nilaimutlak = fabs(angka)
    return nilaimutlak

def main():
    a, b, c, d = map(int, input("").split())
    hasil = mutlak(hitung(a, c)) + mutlak(hitung(b, d))
    print (int(mutlak(hasil)))

if __name__ == "__main__":
    main()